### ISSUE TITLE
[a11y] drag handle missing roles

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161557]
When using this widget https://api.flutter.dev/flutter/widgets/ReorderableList-class.html,  the semantics role should have the following

| OS | role |
|--------|--------|
| web |- |
| ios | - |
| macos |  NSAccessibilityHandleRole |
| windows | - |
| android | - | 

When using this widget https://api.flutter.dev/flutter/material/BottomSheet-class.html with showDragHandle, the semantics role should have the following

| OS | role |
|--------|--------|
| web | - |
| ios | - |
| macos | - |
| windows | - |
| android | BottomSheetDragHandleView |
